checker/tests/index/SameLenNewArrayWithSameLength.java

```diff
@@ -0,0 +1,7 @@
+import org.checkerframework.checker.index.qual.*;
+
+class SameLenNewArrayWithSameLength {
+    public void m1(int[] a) {
```

Comment:
Please add test cases where `a` is the same length of other arrays (besides `b`).